includes/class-post-type-offer.php

```diff
@@ -0,0 +1,1289 @@
+<?php
+namespace bedIQ;
+/**
+ * Class Post_Type_Offer
+ */
+class Post_Type_Offer extends Post_Type {
+    /**
+     * Constructor for Post_Type_Offer
+     */
+    function __construct() {
+        add_action( 'init', array( $this, 'register_post_type' ) );
+        add_action( 'init', array( $this, 'register_taxonomy' ) );
```

Comment:
Remove this `init` calls and move them to the main plugin file as instruction given in  `Insert_Term ->register_taxonomy()`.